package.json

```diff
@@ -10,14 +10,15 @@
     "url": "https://github.com/ipython/ipywidgets.git"
   },
   "scripts": {
+    "postinstall": "npm run bower",
     "build": "npm run css && npm run buildtests",
     "bower": "bower install --config.interactive=false",
     "precss": "rimraf ./ipywidgets/static/widgets/css/",
     "css": "node ipywidgets/build_css.js",
     "prebuildtests": "rimraf ./ipywidgets/tests/bin/tests",
     "buildtests": "tsc ./ipywidgets/tests/**/*.ts --outDir ./ipywidgets/tests/bin -d -m commonjs -t ES5",
-    "prepublish": "npm run bower && npm run build",
-    "publish": "npm run publish:pypi",
```

Comment:
I understood prepublish was ran at install time for local installs.  I thought it ran after the dev-dependencies were installed, so executing bower and build is okay.  Did you find otherwise?

With the new setup, if I `git clone`, `cd`, and then `npm install`, the build step no longer is ran for me.